src/tokenize.js

```diff
@@ -253,7 +253,7 @@ function _tokenize(filename, readline, encoding, yield_) {
     var Number_ = group(Imagnumber, Floatnumber, Intnumber);
     var PseudoToken = Whitespace + group(PseudoExtras, Number_, Funny, ContStr, Name);
 
-    const PseudoTokenRegexp = new RegExp(PseudoToken);
+  const PseudoTokenRegexp = new RegExp(PseudoToken, 'u');
```

Comment:
We shouldn't use the 'u' flag here since we still support older browsers.
To make this work correctly for the failed example you should adjust the `Name` category from `\w+` to something similar to the link you provided to stackoverflow in the description.

Reply:
Thank you! I'll have a go at it.